discovery-provider/src/monitors/monitors.py

```diff
@@ -0,0 +1,177 @@
+import json
+from src.monitors.database import \
+    get_database_connection_info, \
+    get_database_connections, \
+    get_database_liveness, \
+    get_database_size
+from src.monitors.memory import \
+    get_total_memory, \
+    get_used_memory
+from src.monitors.filesystem import \
+    get_filesystem_size, \
+    get_filesystem_used
+from src.monitors.network import \
+    get_received_bytes_per_sec, \
+    get_transferred_bytes_per_sec
+from src.monitors.redis import \
+    get_redis_num_keys, \
+    get_redis_used_memory, \
+    get_redis_total_memory
+from src.utils import redis_connection
+
+redis = redis_connection.get_redis()
+
+
+MONITORING_REDIS_PREFIX = 'monitoring'
+
+"""
+    List of all monitors to run, containing
+    Args:
+    name: string The name of the metric (used to key into redis )
+    func: function The function to compute the metric. Takes in kwargs
+        db = The singleton db instance
+        redis = The singleton redis instance
+    ttl: int (optional) TTL in seconds for how long a cached value is good for, min 60s
+    type: int (optional) type that the value should be parsed to (default is str)
+        Options are bool, int, float, str, json
+"""
+
+DATABASE_LIVENESS = {
```

Comment:
are these names 1:1 to the keys used in the content node?

Reply:
yep! (minus snake vs camel case)